src/controller/ai_controller.c

```diff
@@ -1128,16 +1128,18 @@ int ai_har_event(controller *ctrl, har_event event) {
 
             if(event.type == HAR_EVENT_LAND_HIT_PROJECTILE) {
                 // we hit with a projectile
-                chain_consider_tactics(ctrl, (int[]){TACTIC_FLY, TACTIC_TURTLE, TACTIC_CLOSE, TACTIC_SHOOT});
+                int tacs[] = {TACTIC_FLY, TACTIC_TURTLE, TACTIC_CLOSE, TACTIC_SHOOT};
+                chain_consider_tactics(ctrl, tacs, N_ELEMENTS(tacs));
             } else {
                 // we hit with a HAR attack
-                chain_consider_tactics(ctrl, (int[]){TACTIC_QUICK, TACTIC_TRIP, TACTIC_GRAB, TACTIC_PUSH, TACTIC_CLOSE,
-                                                     TACTIC_SHOOT, TACTIC_TURTLE, TACTIC_SPAM});
+                int tacs[] = {TACTIC_QUICK, TACTIC_TRIP,  TACTIC_GRAB,   TACTIC_PUSH,
+                              TACTIC_CLOSE, TACTIC_SHOOT, TACTIC_TURTLE, TACTIC_SPAM};
+                chain_consider_tactics(ctrl, tacs, N_ELEMENTS(tacs));
             }
-            break;
+        } break;
```

Comment:
These added ending braces are after the break in the previous such commit, should put them the same way on both. I don't care which way.

Reply:
Fix'd them all, I think.